appcues/src/main/java/com/appcues/ui/composables/CompositionLocals.kt

```diff
@@ -49,10 +56,26 @@ internal val LocalLogcues = staticCompositionLocalOf<Logcues> { noLocalProvidedF
 
 internal val LocalExperienceStepFormStateDelegate = compositionLocalOf { ExperienceStepFormState() }
 
-internal val LocalStackScope = compositionLocalOf { COLUMN }
+internal val LocalStackScope = compositionLocalOf<StackScope> { ColumnStackScope(null, 0) }
+
+internal sealed class StackScope(private val childrenCount: Int) {
+
+    private var childrenSizes = hashMapOf<UUID, Int>()
+    private val _greaterHeight = mutableStateOf(0)
+    val greaterHeight: State<Int> = _greaterHeight
```

Comment:
should this be `greatestSize`? meaning height or width depending on the type of scope? Should it be a Double?

Reply:
it should be greatestSize, but its a Int, its actually pixel from a IntSize object